src/Shell/Task/UnloadTask.php

```diff
@@ -58,15 +63,19 @@ public function main($plugin = null)
      */
     protected function _modifyBootstrap($plugin)
     {
-        $finder = "/\nPlugin::load\((.|.\n|\n\s\s|\n\t|)+'$plugin'(.|.\n|)+\);\n/";
+        $finder = "@\nPlugin::load\((.|.\n|\n\s\s|\n\t|)+'$plugin'(.|.\n|)+\);\n@";
```

Comment:
Yeah that might be out of scope of this PR, though.
Maybe you can fix up the regexp improvements afterwards?